gc/stats/ScavengerStats.hpp

```diff
@@ -98,6 +98,17 @@ class MM_ScavengerStats
 	uintptr_t _copyScanUpdates;
 #endif /* J9MODRON_TGC_PARALLEL_STATISTICS */
 
+	/* Stats Used Specifically for Adaptive Threading */
+	uint64_t _workerScavengeStartTime; /**< Timestamp taken when worker starts the scavenge task */
+	uint64_t _workerScavengeEndTime; /**< Timestamp taken when worker completes the scavenge task */
```

Comment:
I wish there are simple called _start/_endTime (this is Thread local and Scavenge specific class already, and workerScavenge is redundant). 

But I understand that those already exist, although they are somewhat poorly used - not as thread local ones, but to record total cycle duration, done only my main thread. Ideally, the existing ones should be moved somewhere else (for example in Collector class), but that would required some significant changes (how Verbose GC obtains them).

For now, this is good, but something to think about. At least, please, add a comment to existing ones why they are used for.